### PR TITLE
Profile page stub

### PR DESCRIPTION
**Pull request by aviars**

### CODE REVIEW

File: apps/home/views.py [hunk 1]
```diff
@@ -13,9 +14,13 @@
 @login_required
 def user_profile(request, subject=None):
     if not subject:
+        # Looking at your own record.
         user = request.user
     else:
         up = get_object_or_404(UserProfile, subject=subject)
+        # Check permission view other profiles.
+        if not request.user.has_perm('can_view_profile_another_user'):
```
Comment (whytheplatypus) [inline apps/home/views.py]: should be something like `request.user.has_perm('userprofile.view')` it might be `accounts.userprofile.view`.

Comment (aviars) [reply apps/home/views.py]: fixed this. correct reference is `accounts.view_userprofile`

File: apps/accounts/models.py [hunk 2]
```diff
@@ -57,6 +82,10 @@ def doc_oidc_format(self):
         od['num'] = self.value
         return od
 
+    class Meta:
+        permissions = (
+            ("can_change_individual_identifier", "Can change individual identifier"),)
```
[the remark this answers is not in the document]
Comment (aviars) [reply apps/accounts/models.py]: fixed.

File: apps/accounts/models.py [hunk 3]
```diff
@@ -74,6 +103,10 @@ class OrganizationIdentifier(models.Model):
     def __str__(self):
         return self.value
 
+    class Meta:
+        permissions = (
+            ("can_change_organization_identifier", "Can change organization identifier"),)
```
[the remark this answers is not in the document]
Comment (aviars) [reply apps/accounts/models.py]: fixed.

File: apps/accounts/models.py [hunk 4]
```diff
@@ -117,6 +150,10 @@ def formatted_address(self):
         od['country'] = self.country
         return od
 
+    class Meta:
+        permissions = (
+            ("can_change_address", "Can change address"),)
```
[the remark this answers is not in the document]
Comment (aviars) [reply apps/accounts/models.py]: fixed.

**Pull request merged**